projects/igniteui-angular/src/lib/grids/filtering/base/grid-filtering-row.component.html

```diff
@@ -51,7 +51,7 @@
         (onSelection)="onDateSelected($event)"
         (onClosed)="datePickerClose()">
         <ng-template igxDatePickerTemplate let-openDialog="openDialog">
-            <igx-input-group #inputGroup type="box" [displayDensity]="'compact'" (focusout)="onInputGroupFocusout()">
+            <igx-input-group #inputGroup type="box" [displayDensity]="displayDensity" (focusout)="onInputGroupFocusout()">
```

Comment:
This should probably also be fluent.
```suggestion
            <igx-input-group #inputGroup type="fluent" [displayDensity]="displayDensity" (focusout)="onInputGroupFocusout()">
```